docs/api/structures/usb-device.md

```diff
@@ -1,17 +1,35 @@
 # USBDevice Object
 
+* `configuration` Object (optional) - A [USBConfiguration](https://developer.mozilla.org/en-US/docs/Web/API/USBConfiguration) object containing information about the currently selected configuration of a USB device.
+  * `configurationValue` Integer - the configuration value of this configuration.
+  * `configurationName` String - the name provided by the device to describe this configuration.
+  * `interfaces` Object[] - An array of [USBInterface](https://developer.mozilla.org/en-US/docs/Web/API/USBInterface) objects containing information about an interface provided by the USB device.
+    * `interfaceNumber` Integer - the interface number of this interface.
+    * `alternate` Object - the currently selected alternative configuration of this interface.
+      * `alternateSetting` Integer - the alternate setting number of this interface.
+      * `interfaceClass` Integer - the class of this interface. See [USB.org](https://www.usb.org/defined-class-codes) for class code descriptions.
+      * `interfaceSubclass` Integer - the subclass of this interface.
+      * `interfaceProtocol` Integer - the protocol supported by this interface.
+      * `interfaceName` String (optional) - the name of the interface, if one is provided by the device.
+      * `endpoints` Object[] - an array containing instances of the [USBEndpoint interface](https://developer.mozilla.org/en-US/docs/Web/API/USBEndpoint) describing each of the endpoints that are part of this interface.
+        * `endpointNumber` Integer - this endpoint's "endpoint number" which is a value from 1 to 15.
+        * `direction` String - the direction in which this endpoint transfers data - can be either 'in' or 'out'.
+        * `type` String - the type of this endpoint - can be either 'bulk', 'interrupt', or 'isochronous'.
+        * `packetSize` Number - the size of the packets that data sent through this endpoint will be divided into.
```

Comment:
We don't have a unified standard in the API docs for this yet (ref https://github.com/electron/docs-parser/pull/144) but can we stick to either `String` or `string` for the raw type within this structure?

Reply:
There's a `no-wrapper-object-types` [rule](https://typescript-eslint.io/rules/no-wrapper-object-types) for ESLint that encourages use of `string` over `String` (as well as `number` over `Number`, etc.)

Seems worthwhile to conform to that recommendation even if we don't have an easy way to enforce it directly.

Reply:
The referenced `electron/docs-parser` PR will enforce it, albeit not in the most user friendly way (by the time the error is thrown, things like line number info aren't available).